src/libraries/Microsoft.PowerFx.Core/Types/Enums/EnumStoreBuilder.cs

```diff
@@ -13,8 +13,9 @@ namespace Microsoft.PowerFx.Core.Types.Enums
     /// <summary>
     /// Static class used to store built in Power Fx enums.
     /// </summary>
-    internal sealed class EnumStoreBuilder
+    internal sealed class EnumStoreBuilder : IEnumStore
     {
+        // These are used by Power Apps 
         #region Default Enums
         internal static IReadOnlyDictionary<string, string> DefaultEnums { get; } =
```

Comment:
The fact that the TValue is string means that we're re-parsing the enum specs constantly.  If I don't cache, it's a huge 6x perf hit.

Reply:
Is there any way to avoid "we're re-parsing the enum specs constantly"? 
Not just improving by 6x but resolving it all.